### PR TITLE
Run Task: Use DevEnv

### PR DESCRIPTION
**Pull request by CyborgPotato**
Run Task fails on Windows occasionally due to error 87 in creating process, this can be avoided, and manually entering env can be avoided by instead running the executable via meson's devenv

### CODE REVIEW

**Comment by tristan957** [comment]
@xclaesse can you take a look at this and give it some thought? Seems like a good idea.

**Comment by CyborgPotato** [comment]
@tristan957 @xclaesse Just wanting to check in if any opinion on this change

**Comment by tristan957** [comment]
I'm sorry for letting this sit, and I will try to get to it as soon as I can. Can you fix the CI while you're here?

**Comment by CyborgPotato** [comment]
@tristan957 Of course, i fixed the prettier issue and have rebased my fork. Thanks again for considering the PR

File: src/tasks.ts [hunk 1]
```diff
@@ -44,9 +43,8 @@ function createRunTask(t: Target, targetName: string) {
     },
     `Run ${targetDisplayName}`,
     "Meson",
-    new vscode.ProcessExecution(t.filename[0], {
-      cwd: workspaceState.get<string>("mesonbuild.sourceDir")!,
-      env: getEnvDict(),
```
Comment (xclaesse) [inline src/tasks.ts:49]: That means you can also delete getEnvDict function and _envDict related code in utils.ts.

**Comment by xclaesse** [comment]
The intention is clearly to run those exe in devenv, I'm not sure why I haven't done it your way in the first place. Loading the generated env file into a dict seems weird in retrospect. LGTM.

File: src/tasks.ts (hunk 1 above)
Comment (CyborgPotato) [reply src/tasks.ts:49]: Whoops: makes sense, they have been removed :+1:

**Comment by tristan957** [comment]
@xclaesse I'll let you hit the merge button when you're happy.

**Comment by tristan957** [comment]
Can you rebase this, and then I can merge it for the next release?

**Comment by CyborgPotato** [comment]
@tristan957 Thank you! Sorry for the delay, I have rebased the change

**Pull request merged**